openfisca_us/variables/gov/irs/credits/cdcc/cdcc_rate.py

```diff
@@ -0,0 +1,30 @@
+from openfisca_us.model_api import *
+
+
+class cdcc_rate(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "CDCC credit rate"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.law.cornell.edu/uscode/text/26/21#a_2"
```

Comment:
Reading through this, doesn't there need to be some floor/ceiling logic to reduce by integer percentages? Seems like the current code would overly smooth it out.

Reply:
Remind me, am I reading this wrong again? The 'fraction thereof' pattern.

Reply:
Ah no I remember, the 'fraction thereof' refers to the income, not the reduction.